src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 1;
+
+  return x => {
+    return y => {
+      if (count > 3 && count % 2 === 0) {
+        count++;
+
+        return 'Bzzz... Error!';
+      } else {
+        count++;
+
+        return x + y;
+      }
```

Comment:
no need for if else here
increment count only once inside the first inner function

Reply:
ok, i`ll move count in first function  and replace if/else with ternary operator